openlr/road_type_checkers.cpp

```diff
@@ -0,0 +1,56 @@
+#include "openlr/road_type_checkers.hpp"
+
+#include "indexer/classificator.hpp"
+
+namespace openlr
+{
+// TrunkChecker ------------------------------------------------------------------------------------
+TrunkChecker::TrunkChecker()
```

Comment:
All the such checkers are grouped in indexer/ftypes_matcher.hpp. Probably it's better to move this checker to indexer and implement them using ftypes::BaseChecker?